deployments/builder-packet-c1-small-x86-2/l73iy1yk17sgw2dsibmxjwxnr049xjmy-installed.nix

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
why does this file have a hash in it's name?

Reply:
Because we copy the configuration from the imported packet.net machine verbatim.